controllers/provider-aws/charts/provider-aws/templates/configmap-logging.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ include "name" . }}-logging-config
+    namespace: garden
+    labels:
+        extensions.gardener.cloud/configuration: logging
+data:
+  filter-kubernetes.conf: |-
+    [FILTER]
+        Name                parser
+        Match               kubernetes.cloud-controller-manager*cloud-controller-manager*
+        Key_Name            log
+        Parser              kubeapiserverParser
+        Reserve_Data        True
+
+    [FILTER]
+        Name                parser
+        Match               kubernetes.machine-controller-manager*machine-controller-manager*
+        Key_Name            log
+        Parser              kubeapiserverParser
+        Reserve_Data        True
+
+    [FILTER]
+        Name                parser
+        Match               kubernetes.*lb-readvertiser*lb-readvertiser*
+        Key_Name            log
+        Parser              lbReadvertiserParser
+        Reserve_Data        True
+
+    [FILTER]
+        Name record_modifier
+        Match *cloud-controller-manager*
+        Record type user
+
+  parsers.conf: |-
```

Comment:
It should only be deployed if the seed provider is `aws`: https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-aws/charts/provider-aws/templates/daemonset-custom-mtu.yaml#L1